README.md

```diff
@@ -1,70 +1,154 @@
+
+_coming soon!_
 
-**Private Kit WhitePaper:** [Apps Gone Rogue: Maintaining Personal Privacy in an Epidemic](https://drive.google.com/file/d/1nwOR4drE3YdkCkyy_HBd6giQPPhLEkRc/view?usp=sharing)
+### Downloads for Private Kit (technology beta)
 
-**Downloads for COVID Safe Paths:** _coming soon!_
+[Google Play](https://play.google.com/store/apps/details?id=edu.mit.privatekit) | [Apple Store](https://apps.apple.com/us/app/private-kit-prototype/id1501903733)
 
-**Downloads for Private Kit (technology beta):** [Google Play](https://play.google.com/store/apps/details?id=edu.mit.privatekit) | [Apple Store](https://apps.apple.com/us/app/private-kit-prototype/id1501903733)
+<br />
 
 # Development Overview
 
-This is a React Native app version 61.5
+![Android and iOS build on MacOS](https://github.com/tripleblindmarket/covid-safe-paths/workflows/Android%20and%20iOS%20build%20on%20MacOS/badge.svg)
+
+_Safe Paths_ is a built on [React Native](https://reactnative.dev/docs/getting-started) v0.61.5
+
+## Contributing
+
+Read the [contribution guidelines](CONTRIBUTING.md).
 
 ## Architecture
 
-Please refer to `docs/Private Kit Diagram.png` for a basic overview on the sequencing of generalized events and services that are used by Safe Paths.
+View the [architecture diagram](docs/Private_Kit_Diagram.png) for a basic overview on the sequencing of generalized events and services that are used by Safe Paths.
 
 ## Developer Setup
 
-Run the ```dev_setup.sh``` (Linux/MacOS) or ```dev_setup.bat``` (Windows) for needed tools.  It is assumed that Android Studio and/or xcode (on macOS) is set up and configured correctly to run applications in the simulator.  For Android Studio, the dev_setup script can help you, but you still may need to configure at least one android virtual device (avd) from within Android Studio first before you can run the app.
+First, run the appropriate setup script for your system. This will install relevant packages, walk through Android Studio configuration, etc.
+
+**Note:** You will still need to [configure an Android Virtual Device (AVD)](https://developer.android.com/studio/run/managing-avds#createavd) after running the script.
+
+#### Linux/MacOS
+
+```
+dev_setup.sh
+```
+
+#### Windows
+
+```
+dev_setup.bat
+```
 
 ## Running
 
-```yarn``` must be installed for this project, as it does a better job installing dependendies across platforms.  The dev_setup script above should have done that for you.
+`yarn` must be installed for this project, as it does a better job installing dependencies across platforms. The dev_setup script above should have done that for you.
 
 ### Install modules to the correct locations
 
-1. ```yarn install``` (always do this after a new clone or checkout)
-2. ```yarn install:pod``` (additional step for macOS if you want to do an iOS build)
+1. `yarn install` (always do this after a new clone or checkout)
+2. `yarn install:pod` (additional step for macOS if you want to do an iOS build)
```

Comment:
`yarn run-ios` will run this automatically, perhaps we can suggest just the `yarn run-ios` and `run-android` commands here?